src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -403,10 +437,7 @@ public String generateRoutingKey(JsonObject eiffelMessage, String tag, String do
      */
     public static Source setSerializerGav(Source source) {
         source = source == null ? new Source() : source;
-        Serializer serializer = source.getSerializer() == null ? new Serializer() : source.getSerializer();
-        serializer.setGroupId(semanticsGAV.getGroupId());
-        serializer.setArtifactId(semanticsGAV.getArtifactId());
-        serializer.setVersion(semanticsGAV.getVersion());
+        String serializer = source.getSerializer() == null ? new String("pkg:maven/"+semanticsGAV.getGroupId()+"/"+semanticsGAV.getArtifactId()+"@"+semanticsGAV.getVersion()) : source.getSerializer();
```

Comment:
we dont use Gav right
directly get semantics information as purl format from manifest.